core/shared/src/main/scala/zio/internal/FiberRuntime.scala

```diff
@@ -279,7 +279,11 @@ class FiberRuntime[E, A](fiberId: FiberId.Runtime, fiberRefs0: FiberRefs, runtim
           case trampoline: Trampoline =>
             trampolines = trampolines + 1
 
-            if (!trampoline.forceYield && (trampolines < FiberRuntime.MaxTrampolinesBeforeYield)) {
+            if (
```

Comment:
This is "trampoline yielding". There is another place inside `runLoop` where there is yielding: operation yielding. Both places have to be modified in order to completely disable cooperative yielding.

Reply:
Ah yes, you are right, forgot I unified those two branches!